tests/jsTests/indexSpec.js

```diff
@@ -1,5 +1,9 @@
-describe("Tests for show error", function() {
-	var msg = 'this is a test message';
+/**
+ * Suite : Tests for showing error on incorrect field entry
+ * Description : This is a test suite for showing error in case of an invalid field entry
+ */
+describe("Tests for showing error on incorrect field entry", function() {
+	var msg = 'This is a test message for showing error';
```

Comment:
@fatimarafiqui they need not be generic messages, you can change them as per the need. I agree with @Anjali05 that they should be specific to the error, instead of generic.